src/Modules/OrchardCore.Commerce/Views/StripeApiSettings.Edit.cshtml

```diff
@@ -11,9 +11,12 @@
 
 <div class="mb-3" asp-validation-class-for="SecretKey">
     <label asp-for="SecretKey">@T["Secret Key"]</label>
-    <input asp-for="SecretKey" class="form-control" type="text" autocomplete="off" />
+    <input asp-for="SecretKey" class="form-control" type="password" autocomplete="off" />
     <span asp-validation-for="SecretKey"></span>
-    <span class="hint">@T["Your Stripe Secret API key."]</span>
+    <span class="hint">
+        @T["Your Stripe Secret API key. The Stripe Secret API key will not be shown and the field will be empty after" +
+            " saving it, for security reasons."]
```

Comment:
You mustn't split the string constant passed to T, because then [poextractor](https://github.com/OrchardCoreContrib/OrchardCoreContrib.PoExtractor) can't extract it. (I've just checked.)